postr/praw_reddit.py

```diff
@@ -1,7 +1,7 @@
 from typing import Any
 
 import config
-import praw
+import praw   # pylint: disable=wrong-import-order
```

Comment:
you should fix the import order instead of ignoring it. Pretty sure one of our hooks will automatically fix it. If pylint is conflicting with autopep8 on import order then let's fix that at a global config level

Reply:
The import order is recursive. The hook auto fixes it, then complains about the order again.